airbyte-integrations/connectors/source-github-singer/setup.py

```diff
@@ -31,7 +31,7 @@
     author_email="contact@airbyte.io",
     packages=find_packages(),
     install_requires=[
-        "tap-github @ https://github.com/airbytehq/tap-github/tarball/v1.9.3-airbyte",
+        "tap-github @ https://github.com/vitaliizazmic/tap-github/tarball/handling_teams_stream",
```

Comment:
Could you make sure to tag the release in https://github.com/airbytehq/tap-github/pull/2 as `v1.9.5-airbyte`? You'd just need to: 

1. Merge the fork pull request
2. checkout master locally, make sure it is up to date
3. `git tag <version in setup.py>`
4. `git push --tags`